src/create/command.ts

```diff
@@ -141,6 +144,29 @@ export async function twCreate(options: any) {
       }
     }
 
+    console.log(projectType);
+    console.log(baseContract);
+    // Select base contract
+    if (projectType === "contract" && !baseContract) {
+      const res = await prompts({
+        type: "select",
+        name: "baseContract",
+        message: "What base contract do you want to use?",
+        choices: [
+          { title: "Empty Contract", value: "" },
+          { title: "ERC-721 Base", value: "ERC721Base" },
+          { title: "ERC-721 + Signature Mint", value: "ERC721DelayedReveal" },
```

Comment:
All 3 of these options have the same `ERC721DelayedReveal` value but they have different titles

Reply:
that is a mistake, they should map correctly